### ISSUE TITLE
Party cmatrix

### ISSUE THREAD

**Issue by Privth** [abishekvashok/cmatrix#86]
### I am submitting a feature

Hi, I accidentaly find this bug/feature? 

I use alias to ruby-gem colorls:

alias ll='colorls -lA --sd --group-directories-first'
alias ls='colorls --group-directories-first'

### Problem or Feature in Brief

https://www.youtube.com/watch?v=Xpx1l1yYgbs&feature=youtu.be

### Would you like to work on it?

no

**Comment by abishekvashok** [comment]
It's a bug rather than an intentional feature or Easter egg.

**Comment by wadkar** [comment]
A very cool ~bug~ feature, if you ask me :-)

**Comment by abishekvashok** [comment]
Shall we add this to docs then?

**Comment by wadkar** [comment]
Yeah, we can but not sure how to describe it. When I tried unaliased `ls | cmatrix` it locked my terminal and messed it up after I `pkill cmatrix` from another tmux pane. I think the ~bug~ feature is that cmatrix will take piped input as `argv`s and change its mode constantly?

It's interesting when you `gem install colorls` and pipe `alias ls='colorls -lA --sd --group-directories-first'` output to cmatrix `ls | cmatrix`. The terminal doesn't get locked up and it works fine when you quit with `Ctrl-c` (`q` takes a looooooong time for it to stop).

I can send in a PR if you like and we can take it from there.

**Comment by abishekvashok** [comment]
Sure do it

**Comment by space-pagan** [comment]
I'll be patching this out. It doesn't make sense for cmatrix to take in arguments past the ones defined.
